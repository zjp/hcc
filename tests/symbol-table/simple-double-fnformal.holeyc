bool glob;
int main(int b, int b){
	int a;
	glob = a;
}
