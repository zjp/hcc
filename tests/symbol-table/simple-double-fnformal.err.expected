FATAL [2,21]: Multiply declared identifier
Name Analysis Failed
